client/src/main/java/org/eclipse/hono/client/CommandClient.java

```diff
@@ -106,4 +106,8 @@
      * @see RequestResponseClient#setRequestTimeout(long)
      */
     Future<Void> sendOneWayCommand(String command, String contentType, Buffer data, Map<String, Object> properties);
+
+    Future<Void> sendCommandWithTenantScopedResponse(String command, Buffer data);
+
+    Future<Void> sendCommandWithTenantScopedResponse(String command, String contentType, Buffer data, Map<String, Object> properties);
```

Comment:
this method should be removed with the advent of `AsyncCommandClient`, shouldn't it?

Reply:
ups, so sorry. Something went terribly wrong with my local merging.